### ISSUE TITLE
entitlement check failed and entitlement not found

### ISSUE THREAD

**Issue by sharonof** [LibreVR/Revive#1168]
I downloaded the games Dead And Buried and Chronos (not official) of a claw I start them through Inject, errors of entitlement check failed and entitlement not found appear.

I decided to buy the game Dead And Buried in official shop Oculus Store, after purchase a game at was started. And unofficial version too.
Did everything that is specified at you in the reference to avoid a mistake https://github.com/LibreVR/Revive/wiki/Troubleshooting.
Nothing helped.
Other games which I get through Oculus Store work well.
Whether it is possible that Oculus will verify authenticity of games?

### Technical Information
*Tell us your specs and versions*
  - Ryzen 7 -1600 :
  - Nvidia Gigabite 1070Ti:
  - DDR4- 16 gb
  - WIN 10 pro: 
  - Vive /

  - Revive (last version)
  - SteamVR version 1.2.9
  - Revive installation version:

### What apps/games are you trying to run?
Dead And Buried 
Chronos 

## Error messages/codes
 entitlement check failed
entitlement not found
Thank you for reporting your issues/feedback/suggestions about Revive! We really appreciate your support!
[ReviveInjector.txt](https://github.com/LibreVR/Revive/files/2795144/ReviveInjector.txt)
[ReviveOverlay.txt](https://github.com/LibreVR/Revive/files/2795145/ReviveOverlay.txt)

**Comment by Dragoboss** [comment]
First thing to always try upon a entitlement error is to open the Oculus desktop client, go to Settings, open the 'Beta' tab and press 'Restart Oculus'.

In most of cases that fixes it. 
Secondly, do you have multiple library locations for Oculus or just one? 
If multiple, Revive only reads the 'Default library location', which may cause these errors. 

Lastly you confuse me by naming 'official Oculus store' (is there an unofficial Oculus store?) and 'the unofficial version' (what do you mean by that....). 

Let me know ;)

**Comment by sharonof** [comment]
Good afternoon.
Checked, I chose default library location С:\Program files\Oculus\Softaware 
I download unofficial versions of games here from this resource. https://the-eye.eu/public/Games/VR/
When I established the game "Lucky Tale" through Oculus store. it began to be started through Revive without error of entitlement check failed as soon as I delete it at myself from library of games, the mistake appears. Perhaps oculus carries out an inspection of the license of games.
